docs/tutorials/teach-atomist-new-skills.md

```diff
@@ -0,0 +1,103 @@
+Software development does not occur solely on the developer's machine any
+longer. It has become a collaborative task opened to all members of the team
+and interactions are constant.
+
+The Atomist Bot you have enrolled to your team strives to be tailored to what
+makes sense to your team. To do so, you can teach the bot new skills 
+automating common tasks your team routinely performs. Those skills are 
+implemented inside Rug command handlers and this tutorial will show you their 
+powerful, and indeed fun, capabilities.
+
+!!! tip "All you need is Rug"
+    These tutorials assume you have a [Rug project][ugpj] which hosts your
+    new [Rug command handlers][rugcmd]. You can easily add them using the 
+    [AddTypeScriptCommandHandler Rug editor][rugeditor] provided by Atomist.
+
+[ugpj]: /user-guide/rug/projects.md
+[rugcmd]: /user-guide/rug/commands.md
+[rugeditor]: https://github.com/atomist/rug-editors#addtypescriptcommandhandler
+
+## Search StackOverflow using the Bot
+
+StackOverflow is one of the biggest venue for developers to collaborate
+together. It is a rather common place to go to and using Atomist you can let
+your team enjoy the service from the team's channel, interacting with each
+other.
+
+Let's see a command handler you could use to query StackOverflow:
+
+```typescript linenums="1"
+{!../../.atomist/handlers/command/SearchStackOverflow.ts!}
+```
+
+Once published to your team, you would be able to make the most of this new
+skill as follows:
+
+```
+@atomist search SO
+```
+
+Upon which the Atomist Bot would prompt your to input your query before going
+its merry way querying StackOverflow. In return, it would disply your the first
+returned items in a nicely formatted Slack message.
+
+Hpw does this work though? You define your Rug command handler and associate
+the intent `search SO`, line 14, which the bot will understand as *call this
+handler now*.
+
+This handler takes a single parameter called `query` holding the user-supplied
+input. Since this does not have a default value, the Atomist Bot will always
+ask for it.
+
+Then, line 23, we define the plan which Atomist will execute as soon as possible
+once the handler returns it. This plan instructs Atomist to call the
+StackOverflow REST API and use the `SendStackOverflowResults` response handler
+on success. That handler is defined on line 43 and simply renders the result
+content from the StackOverflow API call so that it renders nicely in the
```

Comment:
Two uses of `renders`...maybe replace first with `process` or replace second with `looks nice`